src/modules/staking/sora/components/StakeDialog.vue

```diff
@@ -209,9 +224,20 @@ export default class StakeDialog extends Mixins(StakingMixin, mixins.Transaction
     await this.withNotifications(async () => await extrinsic());
     this.$emit('confirm');
   }
+
+  mounted() {
+    const input: HTMLInputElement | null = this.$el.querySelector('.s-input .el-input__inner');
```

Comment:
await this.$nextTick(); // before maybe?

Reply:
yeap